newsletter/model/newsletter_newsletter.py

```diff
@@ -68,7 +68,7 @@ def action_preview(self):
                 'template_id': self.type_id.email_template_id.id,
                 'default_res_id': self.id,
                 'newsletter_res_id': self.env[self.type_id.model.model].search(
-                    safe_eval(self.type_id.domain)).id or False,
+                    safe_eval(self.type_id.domain)).ids[:-1] or False,
```

Comment:
@pedrobaeza it's arbitrary. we just need to pick some record for the preview, and i figured the last object is nicer in some cases. But this totally depends on the sort order of said object and doesn't matter in the end